test/conformance/source/environment.cpp

```diff
@@ -199,6 +202,17 @@ DevicesEnvironment::DevicesEnvironment(int argc, char **argv)
         error = "Could not find any devices associated with the platform";
         return;
     }
+    // Get the argument (test_devices_count) to limit test devices count.
+    u_long count_set = 0;
+    count_set = std::strtoul(argv[2], nullptr, 10);
```

Comment:
check argc before accessing this variable. We should allow this argument to not be present.

Reply:
Done